packages/devextreme-schematics/README.md

```diff
@@ -28,20 +28,21 @@ Some DevExtreme migration schematics require TypeScript to process inline Angula
 - Your project's node_modules
 - The global node_modules
 
-If TypeScript is not found, inline template migration will be skipped and a warning will be shown with resolution attempts and errors.
+If the CLI fails to find TypeScript, inline template migration is skipped and a warning is displayed with resolution attempts and errors.
 
-### How to Fix TypeScript Not Available
+### How to Install TypeScript
 
-1. **Local Project:** Install TypeScript in your project root:
+1. **Local Project:** To install TypeScript in a project, run the following command:
    ```sh
    npm install typescript --save-dev
    ```
-2. **Global CLI:** If you use the CLI globally, also install TypeScript globally:
+2. **Global CLI:** To install TypeScript globally on your machine, run the following command:
    ```sh
    npm install -g typescript
    ```
-3. **npx Usage:** If you use npx, ensure TypeScript is available in your workspace or globally.
-4. **Troubleshooting:**
+
+**Troubleshooting**
+
    - Some npm global installs may not link dependencies as expected. If you see repeated TypeScript resolution errors, try running the CLI from a project where TypeScript is installed locally.
```

Comment:
CLI is "command line interface". We probably shouldn't say "try running the CLI from ....". I think you mean that we run commands, not the CLI itself. 

```suggestion
   - Some npm global installs may not link dependencies as expected. If you experience repeated TypeScript resolution errors, try running CLI commands from a project where TypeScript is installed locally.
```